src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -640,7 +640,7 @@ private void RaiseRenderFrame()
         {
             // Identical to RaiseUpdateFrame.
             if (FramesPerSecond >= double.Epsilon
-                && (VSync == VSyncMode.Off || VSync == VSyncMode.Adaptive && IsRunningSlowly))
+                && (VSync == VSyncMode.Off || VSync == VSyncMode.Adaptive))

```

Comment:
You are no longer using the IsRunningSlowly property, which is core to adaptive vsync.
```cs
    /// <summary>
    /// Use adaptive VSync. If the window IsRunningSlowly, VSync will be disabled. Otherwise,
    /// it will be enabled. If the framerate is uncapped, this is the same as <see cref="On"/>. 
    /// </summary>
```
Am I missing something here?

Reply:
The logic that sets IsRunningSlowly is inside that if block. So if I only enter if it is set, it will never be set at all.

It is still used by SetSwapInterval.

Reply:
Ah ok. As long as you can assure me that IsRunningSlowly is used somewhere to influence the swap interval this is fine with me.

Reply:
Yeah it is. Moreover the return statement below your mentioned code only is executed in the non-adaptive case, as in adaptive mode there can never be more time than needed. If there was, running slow is obviously false and the behavior is like adaptive on. So the logic before didn't make much sense. Testing was successful for all vsync modes but it's good if someone else checks again.